lib/oli/authoring/editing/page_editor.ex

```diff
@@ -330,16 +330,22 @@ defmodule Oli.Authoring.Editing.PageEditor do
                         activity_type_id: activity_type_id,
                         content: content,
                         graded: graded
-                      } ->
+                      } = revision ->
          # To support 'test mode' in the editor, we give the editor an initial transformed
```

Comment:
I guess it makes sense to update the transformed data here since its already part of the editor payload, though I'm not sure it is being used anymore since we nixed the editor test-mode